app/_src/gateway-operator/get-started/konnect/horizontal-autoscaling.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Horizontally autoscale a Data Plane
+content-type: tutorial
+book: kgo-konnect-get-started
+chapter: 4
+---
+
+{% if_version gte:1.2.0 %}
+
+{{ site.kgo_product_name }} can deploy data planes that will horizontally autoscale based on user defined criteria.
+
+If you'd like to configure your data planes based on their average CPU utilization, this is how you can do it.
+
+## Before we begin
+
+In order to perform horizontal autoscaling of data planes, underneath,
+{{ site.kgo_product_name }} uses Kubernetes [`HorizontalPodAutoscaler`][hpa].
+
+{:.note}
+> In order to be able to use `HorizontalPodAutoscaler` in your clusters you'll need to have a [metrics server][metrics_server_github] installed.
+> More info on the metrics server can be found in [official Kubernetes docs][metrics_server].
+
+[metrics_server]: https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#metrics-server
+[metrics_server_github]: https://github.com/kubernetes-sigs/metrics-server
+[hpa]: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+
+## Create a `DataPlane` with horizontal autoscaling enabled
+
+In order to enable horizontal autoscaling for your `DataPlane` you have to specify
+the `spec.deployment.scaling` section to indicate which metrics should be used
+for decision making.
+
+In the example below we'll be using CPU utilization:
+
+```yaml
+apiVersion: gateway-operator.konghq.com/v1beta1
+kind: DataPlane
+metadata:
+  name: horizontal-autoscaling
+spec:
+  deployment:
+    scaling:
+      horizontal:
+        minReplicas: 2
+        maxReplicas: 10
+        metrics:
+        - type: Resource
+          resource:
+            name: cpu
+            target:
+              type: Utilization
+              averageUtilization: 50
+        behavior:
+          scaleDown:
+            stabilizationWindowSeconds: 1
+            policies:
+            - type: Percent
+              value: 100
+              periodSeconds: 10
+          scaleUp:
+            stabilizationWindowSeconds: 1
+            policies:
+            - type: Percent
+              value: 100
+              periodSeconds: 10
+            - type: Pods
+              value: 5
+              periodSeconds: 10
+            selectPolicy: Max
+    podTemplateSpec:
+      spec:
+        containers:
+        - name: proxy
+          image: kong/kong-gateway:{{ site.data.kong_latest_gateway.ee-version }}
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "250m"
+            limits:
+              memory: "1024Mi"
+              cpu: "1000m"
+          # Konnect related environment variables, volumes etc...
+```
+
+> TODO: add CRD reference section link when [KGO#1281][autoscaling_pr] is merged
+> and CRD reference docs are updated ( in reference/custom-resources/1.2.x.md which is yet to be created).
+
+[autoscaling_pr]: https://github.com/Kong/gateway-operator/pull/1281
```

Comment:
I wanted to add a note linking to https://deploy-preview-6710--kongdocs.netlify.app/gateway-operator/1.2.x/reference/custom-resources/#dataplanedeploymentoptions but for some reason this doesn't include the `scaling` section which I have definitely added https://github.com/Kong/docs.konghq.com/pull/6710/files#diff-4d19265cfc4982d86c9005fa2bdd422a19040d4874080edaae1c617df1d9091dR243.

I suppose that's because docs still think that 1.1 is the latest version even though I have added 1.2 in `app/_data/kong_versions.yml`.

Any ideas how to make the new CRD page have the changed (new) content?

Reply:
cc @Kong/docs-maintainers

Reply:
This is solved now, thanks to @mheap 🙇